src/HTMLView.ts

```diff
@@ -0,0 +1,14 @@
+export default class HTMLView {
```

Comment:
I am not sure `HTMLView` should be a class. It seems like it is just a couple of utility methods.

Perhaps it would be better to change `HTMLView` to be a module with two exported functions and change `BookSettings` and `IFrameNavigator` to simply use the functions as needed? Nothing seems to depend on anything being a subclass of `HTMLView`.

If you do want it to remain a class for some reason, I would consider declaring its two methods `static`.